opentelemetry-user-events-logs/Cargo.toml

```diff
@@ -17,14 +17,16 @@ opentelemetry = { workspace = true, features = ["logs"] }
 opentelemetry_sdk = { workspace = true, features = ["logs"] }
 async-trait = { version="0.1" }
 chrono = { version = "0.4", default-features = false, features = ["std"] }
+tracing = { version = "0.1", optional = true }
 
 [dev-dependencies]
 opentelemetry-appender-tracing = { workspace = true }
 tracing = { version = "0.1", default-features = false, features = ["std"] }
 tracing-core = "0.1.31"
-tracing-subscriber = { version = "0.3.0", default-features = false, features = ["registry", "std"] }
+tracing-subscriber = { version = "0.3.0", default-features = false, features = ["env-filter", "fmt", "registry", "std"] }
 microbench = "0.5"
 
 [features]
 spec_unstable_logs_enabled = ["opentelemetry/spec_unstable_logs_enabled", "opentelemetry_sdk/spec_unstable_logs_enabled", "opentelemetry-appender-tracing/spec_unstable_logs_enabled"]
-default = ["spec_unstable_logs_enabled"]
+internal-logs = ["tracing"]
```

Comment:
this need to enable internal-logs in opentelemetry and sdk too.

Reply:
Done.